docs/.vuepress/tools/snippet-transform/snippetTransformer.js

```diff
@@ -0,0 +1,407 @@
+const { parse } = require('espree');
+const { renderTemplate } = require('./templates');
+const { logChange } = require('./helpers/previewLogger');
+const { logger } = require('../utils');
+const { getDefaultFramework } = require('../../helpers');
+const { CategorizedData } = require('./helpers/categorizedData');
+const { Comments } = require('./helpers/comments');
+
+// TODO: IMPORTANT NOTE: detached comments are NOT being transformed (in many cases there's no way of knowing where
+//  to put them in the transformed code)
+
+/**
+ * Variable names for which the expressions that are based on them are automatically moved to the `refExpressions`
+ * section (expressions which are executed after the Handsontable initialization).
+ *
+ * @type {string[]}
+ */
+const REF_VAR_NAMES = [
+  'hot',
+  'hotInstance',
+  'instance',
+  'plugin'
+];
+
+/**
+ * Framework constants.
+ *
+ * @type {{js: string, react: string}}
+ */
+const FRAMEWORKS = {
+  js: 'javascript',
+  react: 'react'
+};
+
+/**
+ * List of frameworks currently supported by the transformer.
+ *
+ * @type {string[]}
+ */
+const SUPPORTED_FRAMEWORKS = [
+  FRAMEWORKS.js,
+  FRAMEWORKS.react
```

Comment:
I think it's better to store the string values for the framework names in a single constant, rather than use them directly in the code. That way, if we ever decide to use `"js"` instead of `"javascript"`, we will have to change it in a single place, not search for all the occurrences scattered across the code.
Why does this bother you?

Reply:
I've thought that `FRAMEWORKS.js/FRAMEWORKS.react` is used only to creating `SUPPORTED_FRAMEWORKS` array. It's not. I'm sorry! It's okey then.